src/KeplerParticles.ts

```diff
@@ -191,6 +191,10 @@ export class KeplerParticles {
     this.shaderMaterial = shader;
     this.geometry = geometry;
     this.particleSystem = new THREE.Points(geometry, shader);
+
+    if (this.get3jsObjects() && this.get3jsObjects()[0]) {
+      this.get3jsObjects()[0].frustumCulled = false;
+    }
```

Comment:
Would it be sufficient to set `this.particleSystem.frustumCulled = false`?

Reply:
hi again ✋ I checked on a few examples and this change (`this.particleSystem.frustumCulled = false`) was sufficient to prevent disappearing of particles